ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -79,16 +80,10 @@ public static void scrub(Map<String, String> attributes, ObjectNode json)
   // see bug 1489560
   private static boolean bug1489560Affected(Map<String, String> attributes, ObjectNode json) {
     final String affectedClientId = "c0ffeec0-ffee-c0ff-eec0-ffeec0ffeec0";
+    Map<String, String> tempAttributes = Maps.newHashMap();
+    ParsePayload.addClientIdFromPayload(tempAttributes, json);
```

Comment:
`addClientIdFromPayload` potentially modifies the json object when normalizing the client ID. I don't think it's a problem here though, but certainly not great to have this side-effect...